src/services/stocks/getStocksData.service.ts

```diff
@@ -0,0 +1,18 @@
+import { StocksResponseType } from "types";
+import { stocksClient } from "../axiosInstance";
+
+const getStocksService = async (
+  period: string,
+  symbol: string
+): Promise<any> => {
+  const client = stocksClient(period, symbol);
+  try {
+    const response = await client.get();
```

Comment:
Use instance in service and add all params in `get()`.
```suggestion
   import { stocksClient } from "../axiosInstance";
const apiKey = process.env.API_KEY;

const getStocksService = async (
  period: string,
  symbol: string
): Promise<Record<string, unknown>> => {
  const client = stocksClient();
  try {
    const response = await client.get(
      `/query?apikey=${apiKey}&function=TIME_SERIES_${period}&symbol=${symbol}`
    );
    return response.data;
  } catch (error) {
    return { error };
  }
};

export { getStocksService };
```